redbiom/admin.py

```diff
@@ -209,7 +256,91 @@ def create_context(name, description):
     ScriptManager.load_scripts()
 
 
-def load_sample_data(table, context, tag=None, redis_protocol=False):
+def _load_axis_data(table, ids, opposite_ids, opposite_id_index, axis_label,
+                    context, batchsize):
+    """Manage the loading of data for a particular axis
+
+    Parameters
+    ----------
+    table : biom.Table
+        The table to obtain data from
+    ids : iterable of str
+        The IDs to obtain data for
+    opposite_ids : iterable of str
+        The IDs of the opposite axis in the table
+    opposite_id_index : dict
+        The index which maps an opposite ID to the index value within
+        the Redis database for the identifier
+    axis_label : str
+        The biom.Table axis label of ids
+    context : str
+        The context to load the data into
+    batchsize : int
+        The number of identifiers to group into a single request
+
+    Notes
+    -----
+    This method does not support non count data.
```

Comment:
Maybe cause its early or the double negative but this took me a second to understand, what about?
```suggestion
    This method only supports count data.
```